arches_project/core/arches/resources.py

```diff
@@ -87,7 +87,8 @@ def close_dialog():
             nodegroup_id = selectedGraph["geometry_node_data"][node_id]["nodegroup_id"]
             selectedNode = {"node_id": node_id, "nodegroup_id": nodegroup_id, "name": selectedGraph["geometry_node_data"][node_id]["name"]}
 
-        geomcoll, geometry_type_dict = geometry_conversion(selectedLayer)
+        geom_convert = Geometries(selectedLayer)
+        geomcoll, geometry_type_dict = geom_convert.geometry_conversion()
```

Comment:
Maybe consider changing `geomcoll` to `geometry_collection` just to be a bit more explicit